zipkin-autoconfigure/ui/src/main/java/zipkin/autoconfigure/ui/ZipkinUiProperties.java

```diff
@@ -22,6 +22,7 @@
   private int queryLimit = 10;
   private int defaultLookback = (int) TimeUnit.DAYS.toMillis(7);
   private String instrumented = ".*";
+  private String logsUrl = "";
```

Comment:
I'd initialize this to null, so that you don't end up with an empty string in the json